src/types/commons.ts

```diff
@@ -66,6 +66,7 @@ export type CountryData = {
 }
 
 export type UrlsConfig = {
+  auth_url?: string
```

Comment:
Was this not in the development branch? 🤔

Reply:
This was a historical change. At the moment of implementing this, `onfido_api_url` wasn't pointed to the API Gateway endpoint (`edge.api.<region>`). Now because it was supported, I've switched back to the old `onfido_api_url`.